LICENSE

```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
       this list of conditions and the following disclaimer in the documentation
       and/or other materials provided with the distribution.
 
-    * Neither the name of the SKA South Africa project nor the names of its
+    * Neither the name of the National Research Foundation (South African Radio Astronomy Observatory) project nor the names of its
```

Comment:
Apart from jenkins complaining about the length of the line, Looks Ok

Reply:
Agree it would be better to split this line.  It isn't a Python file, but still nicer not to have very long lines.